### PR TITLE
Car brand

### PR DESCRIPTION
**Pull request by bltemrah**

### CODE REVIEW

**Review by Marmychok — APPROVED** [review]

File: src/test/java/cydeo/emrah/carBrand.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Car brand
+Ford
+Audi
```
Comment (kayla0408) [inline src/test/java/cydeo/emrah/carBrand.txt]: @bltemrah 
I think you forgot "bmw " as I remember one of the AC says bmw should accept as one of the car brand

Comment (Aitats33) [inline src/test/java/cydeo/emrah/carBrand.txt]: @bltemrah 
I think you forgot Mercedes as I remember one of thr AC says Mercedes should accept as one of the car brand

Comment (bltemrah) [reply src/test/java/cydeo/emrah/carBrand.txt]: @kayla0408 Ooo yes your're right, let me update/add it.

Comment (bltemrah) [reply src/test/java/cydeo/emrah/carBrand.txt]: Hey @Aitats33 
Yes, I will add it. thanks

File: src/test/java/cydeo/emrah/carBrand.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
 Car brand
 Ford
-Audi
\ No newline at end of file
+Audi
+BMW
+Mercedes
```
Comment (Marmychok) [inline src/test/java/cydeo/emrah/carBrand.txt:5]: Nice one

**Review by Marmychok — APPROVED** [review]

**Pull request merged**